Sources/SwiftGRPCNIO/GRPCClient.swift

```diff
@@ -26,18 +26,20 @@ open class GRPCClient {
     port: Int,
     eventLoopGroup: EventLoopGroup
   ) -> EventLoopFuture<GRPCClient> {
+    let multiplexerPromise: EventLoopPromise<HTTP2StreamMultiplexer> = eventLoopGroup.next().makePromise()
+
     let bootstrap = ClientBootstrap(group: eventLoopGroup)
       // Enable SO_REUSEADDR.
       .channelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SO_REUSEADDR), value: 1)
       .channelInitializer { channel in
-        channel.pipeline.add(handler: HTTP2Parser(mode: .client))
-    }
+        let multiplexer = channel.configureHTTP2Pipeline(mode: .client)
+        multiplexer.cascade(to: multiplexerPromise)
+        return multiplexer.map { _ in }
```

Comment:
I guess there's no way to avoid the `.map { _ in }`?

Reply:
I don't think so!